invenio_vocabularies/services/service.py

```diff
@@ -168,3 +169,24 @@ def read_all(self, identity, fields, type, cache=True, **kwargs):
             results = Response(search, results)
 
         return self.result_list(self, identity, results)
+
+
+    def read_many(self, identity, type, ids, fields=None, **kwargs):
+        """Search for records matching the querystring.""" #TODO: Write a better doc string
+
+        es_query = Q("match_all")
+        vocabulary_type = VocabularyType.query.filter_by(id=type).one() 
+        vocab_id_filter = Q('term', type__id=vocabulary_type.id)
+        
+        filters = []
+        for id_ in ids:
+            filters.append(Q('term', **{"id": id_}))
+            
+        query = Q('bool', minimum_should_match=1, should=filters)
```

Comment:
minor: since this is passed as `extra_filter` I would call it `filter`

Reply:
This has been resolved but nor changed nor answered 😅 WDYT about the possible name change?

Reply:
done